src/Components/App/App.js

```diff
@@ -14,21 +15,27 @@ class App extends Component {
     this.state = { 
       movies: {movies: []},
       currentMovie: '',
```

Comment:
We could change this to an empty object to see if that breaks everything since it'll be an object eventually.